pkg/diagnose/firewall.go

```diff
@@ -244,7 +244,7 @@ func verifyConnectivity(localClusterInfo, remoteClusterInfo *cluster.Info, names
 
 	defer sPod.Delete()
 
-	gatewayPodIP, err := getGatewayIP(remoteClusterInfo, localClusterInfo.Name, status)
+	gatewayPodIP, err := getGatewayIP(remoteClusterInfo, localClusterInfo.Submariner.Status.ClusterID, status)
 	if err != nil {
 		return status.Error(err, "Error retrieving the gateway IP of cluster %q", localClusterInfo.Name)
```

Comment:
maybe we should use "localClusterInfo.Submariner.Status.ClusterID" in the log message as well?

Reply:
There are many other places, spanning across other `subctl commands`, where this change is needed. I intend to do this in a separate PR.

Reply:
Yes, they are in the log messages only and can be clubbed with this PR too. But anyways, I'm fine if you want to handle it in a separate PR.